README.md

```diff
@@ -1,35 +1,75 @@
 # AlignmentOfUAVImages
-### Description
-This tool is designed to calculate UAV pixel coordinates in the world coordinate system using the corresponded georeferenced 
-satellite image. The transformation between pair of pictures calculated using key points pairs extracted from both of 
-images. 
+## Description
+This tool is designed to calculate UAV pixel coordinates in the world coordinate system using the corresponded georeferenced
+satellite image. The transformation between pair of pictures calculated using key points pairs extracted from both of
+images.
 ![visualization](https://github.com/AnNyiiik/HWThirdTerm/assets/114094098/b6596732-c49f-47cc-baa8-01f9c97bfd31)
-### Usage
+## Usage
+### 1st tool
 1. Clone the repository running `git clone git@github.com:AnNyiiik/AlignmentOfUAVImages.git` on a terminal.
-1. Add your data to the repository root folder. See how it must be organised here (structure of data folder and its 
+1. Add your data to the repository root folder. See how it must be organised here (structure of data folder and its
    subdirectories should be the same, as well as names of files): [example of data](example_of_data).
 1. Run the command to execute the experiment: `python3 experiment.py PATH_TO_DATA_FOLDER`.
-1. The image with matched key points, hist of reprojection error, file with the reprojection error value are located in 
+1. The image with matched key points, hist of reprojection error, file with the reprojection error value are located in
    each subdirectory:
    * image_matches.png
    * reprojection_error.png
-   * reprojection_error 
- 
+   * reprojection_error
+
     The expectancy value of reprojection error for the whole collection and the diagram are located in the data folder:
    * experimentResults.txt
    * reprojection_error_for_each_pair.png
-### Experimental results
+### 2nd tool
+1. Organise your test data in a following way: [example of data](example_of_data).
+1. run command in the terminal:
+`python make_data_for_unsupervised_deep_homography.py path_to_folder_with_all_the_data path_to_save`
+1. pull docker image: `docker pull anyaaaaaa/unsupervised_deep_homography:version_4`
+1. create a container from image: `docker run -it image_id /bin/bash`
+1. press ctrl+P ctrl+Q to exit
```

Comment:
no need to exit, you can just open new terminal instance and run `docker cp` there